whisper_server/whisper_server.py

```
@@ -17,79 +17,75 @@
+
             return {
                 "text": result.get("text", ""),
                 "processing_time": processing_time,
                 "model": model,
-                "file": file_path
+                "file": file_path,
             }
-            
+
         except requests.exceptions.RequestException as e:
             logger.error(f"API request failed: {str(e)}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)